experimental/emulated_vault/README

```diff
@@ -0,0 +1,23 @@
+The emulated_vault module supplies a HTTP server mimicking RIAK behavior for the usage as traffic-control vault.
+The server may use different type of persistent storage (e.g. file-system), using the proper adaptor.
+Basic requirements: Centos ver >= 7; Python
```

Comment:
What versions of Python are supported (3.5+ only)?

Reply:
Tested on python 2.7. Will update